Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -418,15 +455,55 @@ public void FileGen_FilesOutsideProjectPath()
             var result = new PropertiesFileGenerator(config, logger).GenerateFile();
 
             // Assert
-            AssertExpectedStatus("projectName", ProjectInfoValidity.NoFilesToAnalyze, result);
+            AssertExpectedStatus("project", ProjectInfoValidity.NoFilesToAnalyze, result);
             AssertExpectedProjectCount(1, result);
-
             // No files -> project file not created
             AssertFailedToCreatePropertiesFiles(result, logger);
+            logger.AssertWarningsLogged(1);
+            logger.AssertSingleWarningExists($"File '{fileOutsideRootPath}' is not located under the root directory");
         }
 
         [TestMethod]
-        public void FileGen_SharedFiles()
+        public void GenerateFile_FilesOutOfProjectPath_PrintsCorrectWarnings()
+        {
+            // Files outside the project root should be ignored
```

Comment:
[Nitpick] This comment is a bit weird here, standing on its own. My understanding is that it describes the behavior for the test, in that case I would put it  next to the TestMethod attribute, above the method declaration.